demo/api/src/importer/decorators/field-transformer.decorator.ts

```diff
@@ -0,0 +1,29 @@
+import { ImporterEntityClass } from "../entities/base-target.entity";
+
+export interface FieldTransformerData {
+    key: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    transformFieldCallback: (input: string) => string;
+}
+
+const metadataKey = "fieldTransformer";
+
+/**
+ * Decorator to add a callback function to modify the value before assigning it to the entity property
+ *
+ * @param {function} transformFieldCallback - callback function to modify the value before assigning it to the entity property
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const FieldTransformer = (transformFieldCallback: (input: any) => any) => {
```

Comment:
This appears to be unused in this PR. Could we remove it and add it in a later PR?

Reply:
Yes sure

Reply:
It's now used